### PR TITLE
[TACHYON-767] Add help message into TFsShell

### PR DESCRIPTION
**Pull request by caadecarvalho**
[TACHYON-767] Add help message into TFsShell

### CODE REVIEW

**Comment by caadecarvalho** [comment]
Not sure If I uploaded all the 34 files. These are TfsShell.java, TfsShellCommand.java and all ${SHELL_NAME}Command.java . First time, using this tool, sorry.

File: .gitignore [hunk 1]
```diff
@@ -35,3 +35,5 @@
 dependency-reduced-pom.xml
 target/
 tests.log*
+
+common/src/main/java/tachyon/util/FormatUtils.java
```
Comment (aaudiber) [inline .gitignore]: is this intentional?

File: shell/src/main/java/tachyon/shell/command/CopyFromLocalCommand.java [hunk 2]
```diff
@@ -205,4 +205,9 @@ private void copyPath(File src, TachyonURI dstPath) throws IOException {
   public String getUsage() {
     return "copyFromLocal <src> <remoteDst>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Copies the specified file specified by \"source path\" to the path specified by \"remote path\".";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/CopyFromLocalCommand.java]: `Copies the file specified by <src> to the path specified by <remoteDst>.`

File: shell/src/main/java/tachyon/shell/command/CountCommand.java [hunk 3]
```diff
@@ -91,4 +91,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "count <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays the number of folders and files matching the specified prefix in args.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/CountCommand.java]: `Displays the number of files and directories matching the specified prefix.`

File: shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java [hunk 4]
```diff
@@ -96,4 +96,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "createLineage <inputFile1,...> <outputFile1,...> [<cmd_arg1> <cmd_arg2> ...]";
   }
+
+  @Override
+  public String getDescription() {
+    return "To be defined.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java]: `Creates a lineage.`

File: shell/src/main/java/tachyon/shell/command/DeleteLineageCommand.java [hunk 5]
```diff
@@ -61,4 +61,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "deleteLineage <lineageId> <cascade(true|false)>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Deletes a lineage specified by args. If cascade is specified as true, it deletes dependents lineages.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/DeleteLineageCommand.java]: `Deletes a lineage. If cascade is specified as true, dependent lineages will also be deleted.`

File: shell/src/main/java/tachyon/shell/command/DuCommand.java [hunk 6]
```diff
@@ -78,4 +78,9 @@ private long getFileOrFolderSize(TachyonFileSystem tachyonFS, TachyonURI path)
   public String getUsage() {
     return "du <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays the size of a file or a directory specified by argv.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/DuCommand.java]: `Displays the size of the specified file or directory.`

File: shell/src/main/java/tachyon/shell/command/FileInfoCommand.java [hunk 7]
```diff
@@ -65,4 +65,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "fileinfo <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays the file's all blocks info.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/FileInfoCommand.java]: `Displays all block info for the specified file.`

File: shell/src/main/java/tachyon/shell/command/FreeCommand.java [hunk 8]
```diff
@@ -61,4 +61,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "free <file path|folder path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Constructs a new instance to free the given file or folder from Tachyon.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/FreeCommand.java]: `Frees the given file or directory.`

File: shell/src/main/java/tachyon/shell/command/GetCapacityBytesCommand.java [hunk 9]
```diff
@@ -56,4 +56,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "getCapacityBytes";
   }
+
+  @Override
+  public String getDescription() {
+    return "Gets the capacity of the {@link TachyonFileSystem}.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/GetCapacityBytesCommand.java]: `Gets the capacity of the Tachyon file system.`

File: shell/src/main/java/tachyon/shell/command/GetUsedBytesCommand.java [hunk 10]
```diff
@@ -56,4 +56,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "getUsedBytes";
   }
+
+  @Override
+  public String getDescription() {
+    return "Gets number of bytes used in the {@link TachyonFileSystem}.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/GetUsedBytesCommand.java]: `Gets number of bytes used in the Tachyon file system.`

File: shell/src/main/java/tachyon/shell/command/ListLineagesCommand.java [hunk 11]
```diff
@@ -61,4 +61,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "listLineages";
   }
+
+  @Override
+  public String getDescription() {
+    return "Constructs a new instance to list all the lineages.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/ListLineagesCommand.java]: `Lists all lineages.`

File: shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java [hunk 12]
```diff
@@ -65,4 +65,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "loadMetadata <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Constructs a new instance to load metadata for the given Tachyon path from UFS.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java]: `Loads metadata for the given Tachyon path from the under file system.`

File: shell/src/main/java/tachyon/shell/command/LocationCommand.java [hunk 13]
```diff
@@ -70,4 +70,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "location <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Constructs a new instance to display a list of hosts that have the file specified in args stored.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/LocationCommand.java]: `Displays the list of hosts storing the specified file.`

File: shell/src/main/java/tachyon/shell/command/LsCommand.java [hunk 14]
```diff
@@ -51,4 +51,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "ls <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays information for all directories and files directly under the path specified in args.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/LsCommand.java]: `Displays information for all files and directories directly under the specified path.`

File: shell/src/main/java/tachyon/shell/command/LsrCommand.java [hunk 15]
```diff
@@ -52,4 +52,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "lsr <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays information for all directories and files under the path specified in args recursively";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/LsrCommand.java]: `Displays information for all files and directories under the specified path recursively.`

File: shell/src/main/java/tachyon/shell/command/MkdirCommand.java [hunk 16]
```diff
@@ -66,4 +66,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "mkdir <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Creates a new directory specified by the path in args, including any parent folders that are required.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/MkdirCommand.java]: `Creates the specified directory, including any parent directories that are required.`

File: shell/src/main/java/tachyon/shell/command/MvCommand.java [hunk 17]
```diff
@@ -62,4 +62,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "mv <src> <dst>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Renames a file or directory specified by args.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/MvCommand.java]: `Renames a file or directory.`

File: shell/src/main/java/tachyon/shell/command/PinCommand.java [hunk 18]
```diff
@@ -46,4 +46,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "pin <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Pins the given file or folder in memory (works recursively for a directory). Pinned files are never evicted from memory";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/PinCommand.java]: `Pins the given file or directory in memory (works recursively for directories). Pinned files are never evicted from memory.`

File: shell/src/main/java/tachyon/shell/command/RmCommand.java [hunk 19]
```diff
@@ -65,4 +65,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "rm <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Removes the file specified by argv.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/RmCommand.java]: `Removes the specified file.`

File: shell/src/main/java/tachyon/shell/command/RmrCommand.java [hunk 20]
```diff
@@ -55,4 +55,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "rmr <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Removes the file or directory specified by args (works recursively for directory).";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/RmrCommand.java]: `Removes the specified file or directory (works recursively for directories).`

File: shell/src/main/java/tachyon/shell/command/SetTTLCommand.java [hunk 21]
```diff
@@ -57,4 +57,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "setTTL <path> <time to live(in milliseconds)>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Sets a new TTL value for the file at path both of the TTL value and the path are specified by args.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/SetTTLCommand.java]: `Sets a new TTL value for the file at path.`

File: shell/src/main/java/tachyon/shell/command/TouchCommand.java [hunk 22]
```diff
@@ -60,4 +60,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "touch <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Creates a 0 byte file specified by argv. The file will be written to UnderFileSystem.";
```
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/TouchCommand.java]: `Creates a 0 byte file. The file will be written to the under file system.`

**Comment by aaudiber** [comment]
@caadecarvalho Thanks for the improved shell documentation! I've suggested some rewordings, and I think a change to the .gitignore might have slipped in.

**Comment by aaudiber** [comment]
ok to test

File: .gitignore (hunk 1 above)
Comment (caadecarvalho) [inline .gitignore]: No it wasn't, first time using this tool.

File: shell/src/main/java/tachyon/shell/command/CopyFromLocalCommand.java (hunk 2 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/CopyFromLocalCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/CountCommand.java (hunk 3 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/CountCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java (hunk 4 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java]: I guess that there is a lack of information from inputFile1,inputFileN outputFile1,outputFileN cmd_arg1 cmd_arg2 ...

File: shell/src/main/java/tachyon/shell/command/DeleteLineageCommand.java (hunk 5 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/DeleteLineageCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/DuCommand.java (hunk 6 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/DuCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/FileInfoCommand.java (hunk 7 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/FileInfoCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/FreeCommand.java (hunk 8 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/FreeCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/GetCapacityBytesCommand.java (hunk 9 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/GetCapacityBytesCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/GetUsedBytesCommand.java (hunk 10 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/GetUsedBytesCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/ListLineagesCommand.java (hunk 11 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/ListLineagesCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java (hunk 12 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/LocationCommand.java (hunk 13 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/LocationCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/LsCommand.java (hunk 14 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/LsCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/LsrCommand.java (hunk 15 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/LsrCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/MkdirCommand.java (hunk 16 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/MkdirCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/MvCommand.java (hunk 17 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/MvCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/PinCommand.java (hunk 18 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/PinCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/RmCommand.java (hunk 19 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/RmCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/RmrCommand.java (hunk 20 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/RmrCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/SetTTLCommand.java (hunk 21 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/SetTTLCommand.java]: Nice.

File: shell/src/main/java/tachyon/shell/command/TouchCommand.java (hunk 22 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/TouchCommand.java]: Nice.

**Comment by caadecarvalho** [comment]
Good that you like it! And my bad for .gitignore, it should be excluded!

File: shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java (hunk 4 above)
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/CreateLineageCommand.java]: Fair enough :-) I'd be fine either way

**Comment by aaudiber** [comment]
Looks like there are a few lines over 100 characters and some trailing whitespace:
```
/home/jenkins/workspace/Tachyon-Pull-Request-Builder/shell/src/main/java/tachyon/shell/command/TfsShellCommand.java:52: Trailing whitespace found.
/home/jenkins/workspace/Tachyon-Pull-Request-Builder/shell/src/main/java/tachyon/shell/command/DeleteLineageCommand.java:67: Line is longer than 100 characters (found 106).
/home/jenkins/workspace/Tachyon-Pull-Request-Builder/shell/src/main/java/tachyon/shell/command/PinCommand.java:52: Line is longer than 100 characters (found 137).
/home/jenkins/workspace/Tachyon-Pull-Request-Builder/shell/src/main/java/tachyon/shell/command/LsrCommand.java:58: Line is longer than 100 characters (found 102).
/home/jenkins/workspace/Tachyon-Pull-Request-Builder/shell/src/main/java/tachyon/shell/TfsShell.java:104: Line is longer than 100 characters (found 118).
```

**Comment by caadecarvalho** [comment]
That was tought! Thanks for the feedback info @aaudiber !!

**Comment by aaudiber** [comment]
Please revert changes to `.gitignore` and `common/src/main/java/tachyon/util/FormatUtils.java`.

Also, please try to use more informative commit messages than "[TACHYON-767]" or "Done" in the future. You can modify old commit messages with `git commit --amend` or `git rebase -i HEAD~N` (see https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)

**Comment by caadecarvalho** [comment]
@aaudiber, sorry for the confusion. It's my first time using GitHub and being part of a repository. I executed the deletion but I'm not sure if this is the right thing, I didn't followed what you mentioned because I'm at work and I don't have github installed here. If this isn't correct, I can do it later at home. Regards.

**Comment by caadecarvalho** [comment]
@aaudiber, I guess now it is good for merging.

**Comment by aaudiber** [comment]
No problem! We're getting closer to being able to merge. You can revert the changes to `.gitignore` by running the following from your git repo:
```
git remote add upstream git@github.com:amplab/tachyon.git
git fetch upstream
git checkout upstream/master .gitignore
git commit -m 'Revert changes to .gitignore'
git push origin HEAD:master
```

You'll also need to resolve conflicts with the master branch, which you can do with
```
git merge upstream/master
... resolve conflicts ...
git commit
git push origin HEAD:master
```

File: .gitignore [hunk 23]
```diff
@@ -1,37 +0,0 @@
-**/tachyon-site.properties
```
Comment (uronce-cc) [inline .gitignore]: I think you have unintentionally deleted .gitignore.

File: shell/src/main/java/tachyon/shell/TfsShell.java [hunk 24]
```diff
@@ -101,7 +101,8 @@ public void printUsage() {
     System.out.println("Usage: java TfsShell");
     SortedSet<String> sortedCmds = new TreeSet<String>(mCommands.keySet());
     for (String cmd : sortedCmds) {
-      System.out.println("       [" + mCommands.get(cmd).getUsage() + "]");
+      System.out.format("%-60s%-95s%n", "       [" + mCommands.get(cmd).getUsage() + "]   ",
```
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: the `"      ["` and `"]    "` part would be better to be put into the format string?

File: shell/src/main/java/tachyon/shell/command/CopyFromLocalCommand.java [hunk 25]
```diff
@@ -205,4 +205,9 @@ private void copyPath(File src, TachyonURI dstPath) throws IOException {
   public String getUsage() {
     return "copyFromLocal <src> <remoteDst>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Copies the file specified by <src> to the path specified by <remoteDst>.";
```
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/CopyFromLocalCommand.java]: `Copies the file from local filesystem to Tachyon filesystem`, to be consistent with `copyToLocal`'s description.

File: shell/src/main/java/tachyon/shell/command/CountCommand.java [hunk 26]
```diff
@@ -91,4 +91,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "count <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Displays the number of files and directories matching the specified prefix.";
```
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/CountCommand.java]: "matching the specified prefix" is confusing to me, I may suspect that "/a/b/c*" will match "/a/b/c", "/a/b/cd" like unix glob. 

What about "Displays the total bytes of a file, or the total bytes, number of files and number of directories recursively contained under a directory."?

Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/CountCommand.java]: the class description has this problem too. What do other reviewers think? @aaudiber

File: shell/src/main/java/tachyon/shell/command/FreeCommand.java [hunk 27]
```diff
@@ -61,4 +61,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "free <file path|folder path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Frees the given file or directory.";
```
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/FreeCommand.java]: Be more descriptive like "Removes the file or directory(recursively) from Tachyon memory space."

File: shell/src/main/java/tachyon/shell/command/RmCommand.java [hunk 28]
```diff
@@ -65,4 +65,9 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "rm <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Removes the specified file.";
```
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/RmCommand.java]: "..., use rmr to remove directories."

File: shell/src/main/java/tachyon/shell/command/PinCommand.java [hunk 29]
```diff
@@ -46,4 +46,10 @@ void runCommand(TachyonURI path) throws IOException {
   public String getUsage() {
     return "pin <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Pins the given file or directory in memory (works recursively for directories). "
+      + "Pinned files are never evicted from memory.";
```
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/PinCommand.java]: " ..... unless TTL is set."

File: shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java [hunk 30]
```diff
@@ -65,4 +65,9 @@ public void run(String... args) throws IOException {
   public String getUsage() {
     return "loadMetadata <path>";
   }
+
+  @Override
+  public String getDescription() {
+    return "Loads metadata for the given Tachyon path from the under file system.";
```
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java]: Explain briefly the contents of "metadata"?

**Comment by uronce-cc** [comment]
Thanks for adding descriptions! I left a few comments. 

Please resolve the conflicts. 

Also, I think it looks cleaner to squash the commits into one commit "[TACHYON-767] Add help message into TFsShell".

File: shell/src/main/java/tachyon/shell/command/CountCommand.java (hunk 26 above)
Comment (aaudiber) [inline shell/src/main/java/tachyon/shell/command/CountCommand.java]: The class description isn't user-facing in the same way as the description text, so I'm fine with leaving updating it to another PR

File: shell/src/main/java/tachyon/shell/TfsShell.java (hunk 24 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: I guess would be good to keep those white spaces before and after the command and usage information.

File: shell/src/main/java/tachyon/shell/command/RmCommand.java (hunk 28 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/RmCommand.java]: I guess that this comment should be regarding RmrCommand.java which is bellow. I'll place that.

File: shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java (hunk 30 above)
Comment (caadecarvalho) [inline shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java]: I guess that metadata explanation is better defined at Tachyon docs.

**Comment by caadecarvalho** [comment]
Hi @uronce-cc , I left some commands.

File: shell/src/main/java/tachyon/shell/command/RmCommand.java (hunk 28 above)
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/RmCommand.java]: sgtm

File: shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java (hunk 30 above)
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/command/LoadMetadataCommand.java]: sgtm

File: shell/src/main/java/tachyon/shell/TfsShell.java (hunk 24 above)
Comment (uronce-cc) [inline shell/src/main/java/tachyon/shell/TfsShell.java]: I'm fine with that too :)

**Comment by uronce-cc** [comment]
@caadecarvalho Could you resolve the conflicts and squash commits into one commit as we discussed above? After that, it LGTM.

**Comment by uronce-cc** [comment]
Thanks for adding the help messages!

**Comment by caadecarvalho** [comment]
Guys, I just lost my repository. I'm creating a new pull request in order to fulfill this requirement.

**Pull request closed without merging**